ArduCopter/AP_Arming.cpp

```diff
@@ -495,7 +496,8 @@ bool AP_Arming_Copter::gps_checks(bool display_failure)
 bool AP_Arming_Copter::pre_arm_ekf_attitude_check()
 {
     // get ekf filter status
-    nav_filter_status filt_status = _inav.get_filter_status();
+    nav_filter_status filt_status;
+    _ahrs_navekf.get_filter_status(filt_status);
```

Comment:
We should check the get-filter-status has returned true.  if it hasn't we should fail this check.  This applies to all other uses of this method as well.